src/Methods/Modular_Programming/Hard/Q1/MainTest.java

```diff
@@ -0,0 +1,49 @@
+package Methods.Modular_Programming.Hard.Q1;
+import global.BaseTest;
+import global.variables.*;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class MainTest extends BaseTest{
+    String input = "5";
+
+    public Clause[] testSentence() {
+        provideInput(input);
```

Comment:
Don't we need to put this inside `runMain()`, does this work? It's kinda cool if it does but for now still move it. Also let's move this test off the priority label since it needs input test

Reply:
nooooo it doesn't? I haven't run into any issues writing tests for it. Also what needs to get put into runMain()? The String input? or the provideInput? I didn't put the provideInput into runMain because then it would provide that input any other time I wanted to run the program, even when I didn't want it to provide that specific input.

Reply:
@Opeyem1a Why does this require inputTesting? I don't have any other plans to add more tests and all the tests function at the moment.